Svc/FrameAccumulator/test/ut/FrameAccumulatorTestMain.cpp

```diff
@@ -0,0 +1,50 @@
+// ======================================================================
```

Comment:
Can we look at a coverage report of your UTs?

Reply:
![image](https://github.com/user-attachments/assets/b1b761e2-dae4-48b5-bf3e-b846dcd72a57)

![image](https://github.com/user-attachments/assets/2c569c48-11d5-47fb-beee-72f6555d450b)

![image](https://github.com/user-attachments/assets/597354fa-00cb-4175-91fc-3b024d464002)